packages/backend/src/index.ts

```diff
@@ -1,25 +1,67 @@
+import * as dotenv from 'dotenv'
+
+import { resolve } from 'path'
+import { object, string } from 'zod'
+
+const envPath = resolve(__dirname, '../.env')
+
+const envVars = dotenv.config({ path: envPath })
+
+const envSchema = object({
+  PORT: string(),
+  DB_NAME: string(),
+  DB_USERNAME: string(),
+  DB_PASSWORD: string(),
```

Comment:
We can use the database URL.

Reply:
I removed the 3 entries and replaced them with a DB_URL env variable:

postgres://postgres:password@postgres/testnet